src/iop/clipping.c

```diff
@@ -299,6 +299,10 @@ static inline void adjust_aabb(const float *p, float *aabb)
   aabb[3] = fmaxf(aabb[3], p[1]);
 }
 
+const char *deprecated_msg()
+{
+  return _("this module is deprecated. please use the crop and/or perspective correction modules instead.");
```

Comment:
I agree with that, but I prefer to raise 2 comments to be sure : 
- there's also the orientation module for flipping. but it's not a key feature, so we may prefer to forget it and keep the msg simpler
- do we want to include here a warning about double crop or is it self explanatory ?

Reply:
Good catch ! Now fixed thanks.